src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1150,18 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    
+    /**
+       private static String removePrefixSign(String s, String sign) {
+       return s.replace(sign, "");
+    }
+     */
+    
+    /**
+    * Removes prefix from the given fullString if prefix occurs at the start of the string.
+    */
+    private static String removePrefix(String fullString, String prefix) {
```

Comment:
It seems that the implementation of this method is difference from what is stated by the header comments.
Consider `/pblahblah/pblah` where `/p` is the prefix, will only the prefix at the start be removed?